engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeComponentsTest.scala

```diff
@@ -244,12 +245,320 @@ class RuntimeComponentsTest
       LibraryName("Enso_Test", "Test") ->
       context.pkg.config.componentGroups
         .getOrElse(fail("Unexpected config value.")),
-      LibraryName("Standard", "Base")     -> ComponentGroups.empty,
+      LibraryName("Standard", "Base") ->
+      RuntimeComponentsTest.standardBaseComponents,
       LibraryName("Standard", "Builtins") -> ComponentGroups.empty
     )
     components should contain theSameElementsAs expectedComponents
 
     context.consumeOut shouldEqual List()
   }
 
+  it should "load component groups of standard library" in {
+    val contextId  = UUID.randomUUID()
+    val requestId  = UUID.randomUUID()
+    val moduleName = "Enso_Test.Test.Main"
+
+    val metadata = new Metadata
+
+    val code =
+      """from Standard.Base import all
+        |import Standard.Visualization
+        |
+        |main = "Hello World!"
+        |""".stripMargin.linesIterator.mkString("\n")
+    val contents = metadata.appendToCode(code)
+    val mainFile = context.writeMain(contents)
+
+    // create context
+    context.send(Api.Request(requestId, Api.CreateContextRequest(contextId)))
+    context.receive shouldEqual Some(
+      Api.Response(requestId, Api.CreateContextResponse(contextId))
+    )
+
+    // open file
+    context.send(
+      Api.Request(Api.OpenFileNotification(mainFile, contents))
+    )
+    context.receiveOne shouldEqual None
+
+    // push main
+    context.send(
+      Api.Request(
+        requestId,
+        Api.PushContextRequest(
+          contextId,
+          Api.StackItem.ExplicitCall(
+            Api.MethodPointer(moduleName, "Main", "main"),
+            None,
+            Vector()
+          )
+        )
+      )
+    )
+    val responses =
+      context.receiveAllUntil(
+        context.executionComplete(contextId),
+        timeout = 120
+      )
+    // sanity check
+    responses should contain allOf (
+      Api.Response(requestId, Api.PushContextResponse(contextId)),
+      context.executionComplete(contextId)
+    )
+
+    // check the registered component groups
+    val components = context.languageContext.getPackageRepository.getComponents
+    val expectedComponents = Map(
+      LibraryName("Enso_Test", "Test") ->
+      context.pkg.config.componentGroups
+        .getOrElse(fail("Unexpected config value.")),
+      LibraryName("Standard", "Base") ->
+      RuntimeComponentsTest.standardBaseComponents,
+      LibraryName("Standard", "Test") -> ComponentGroups.empty,
+      LibraryName("Standard", "Database") ->
+      RuntimeComponentsTest.standardDatabaseComponents,
+      LibraryName("Standard", "Visualization") -> ComponentGroups.empty,
+      LibraryName("Standard", "Geo")           -> ComponentGroups.empty,
+      LibraryName("Standard", "Table") ->
+      RuntimeComponentsTest.standardTableComponents,
+      LibraryName("Standard", "Builtins") -> ComponentGroups.empty
+    )
```

Comment:
I think we are testing too much here.

Do we expect to update the tests every time we add a component group to stdlib? Essentially maintaining two lists of component groups (one proper one and a copy in this test)? Seems unnecessary to me.

I think it would be worth to pick just a few selected groups from each library and check that it was properly included, but allowing other not-yet-known additions.

Checking that other libraries are empty seems ok until they start including their own groups from which we could select a few to test too.